ext/nnstreamer/tensor_filter/tensor_filter_tensorflow.cc

```diff
@@ -75,7 +75,7 @@ class TFCore
   int getOutputTensorDim (GstTensorsInfo * info);
   int run (const GstTensorMemory * input, GstTensorMemory * output);
 
-  static std::map < void *, TF_Tensor * >outputTensorMap;
+  std::map < void *, TF_Tensor * >outputTensorMap;
```

Comment:
let's try to hide this as private value, if we decide to change the params in destroy callback.

Reply:
Done.